perfkitbenchmarker/azure/azure_disk.py

```diff
@@ -123,4 +125,7 @@ def Detach(self):
 
   def GetDevicePath(self):
     """Returns the path to the device inside the VM."""
-    return '/dev/sd%s' % chr(ord(DRIVE_START_LETTER) + self.lun)
+    if self.disk_type == disk.LOCAL:
```

Comment:
Is this condition possible? `disk.LOCAL` isn't specified in `DISK_TYPE` above, and there's an assert in `_Create` that the `disk_type` is in `DISK_TYPE`.